### DISCUSSION TITLE
How to get Swag Coupon ?

### DISCUSSION THREAD

**Discussion by UMANG772** [community/community#114019] [GitHub Education]
### Hi everyone,

I'm verified teacher. I need swags for my student's. How to get that coupons anybody can help pls.

**Comment by PATRICKNDIZEYE** [comment]
As a verified teacher, you can acquire swags and coupons for your students by exploring educational discounts, contacting companies directly, joining teacher communities, attending educational events, and utilizing educational platforms. Many companies offer free or discounted resources specifically for educators, and staying engaged with educational communities and events can help you discover available opportunities. Additionally, participating in professional development programs and following companies on social media can provide access to exclusive promotions and incentives. Remember to verify your eligibility and review any terms and conditions associated with the offers before redeeming them.
